kura/org.eclipse.kura.container.provider/OSGI-INF/metatype/org.eclipse.kura.container.provider.ContainerInstance.xml

```diff
@@ -21,28 +21,14 @@
         
         <AD id="container.image" 
             name="Image name" 
-            description="The image the container will be created with. The value will need to be expressed in the form registryURL/imagename in case of a custom registry. 
-            Please fill the following registry credentials in case of pulling from a custom or authenticated Docker hub registry."
+            description="Specifies the image reference that will be used to create this container instance. 
+            The value has to be expressed in the form registryURL/imagename in case of a custom registry. In case the image will be pulled from the official Docker Hub registry, the registryURL can be ignored and 
+            only the imagename can be provided. Default: nginx (the image will be pulled from the Docker Hub registry)"
```

Comment:
I would use the following here: "The value has to be expressed in the form registryURL/imagename. If no registryURL is provided, the official Docker Hub registry will be used as default." which is a more common way to express this kind of concept.

Reply:
Since we're essentially emulatin a `docker pull` command, we should be able to take inspiration from the official documentation: https://docs.docker.com/reference/cli/docker/image/pull/#pull-from-a-different-registry

Reply:
Made a commit, see if it fits